client-impl/simple-client/src/main/java/uk/gov/gchq/palisade/client/SimpleClient.java

```diff
@@ -48,28 +36,20 @@
 import java.util.stream.Stream;
 
 public class SimpleClient<T> {
-    protected final Serialiser<T> serialiser;
-    protected final ResourceService resourceService;
-    protected final AuditService auditService;
-    protected final PolicyService policyService;
-    protected final UserService userService;
-    protected final CacheService cacheService;
-    protected final PalisadeService palisadeService;
+    private final Serialiser<T> serialiser;
 
-    public SimpleClient() {
+    private final ServicesFactory services;
+
+    public SimpleClient(final ServicesFactory services) {
+        Objects.requireNonNull(services);
+        this.services = services;
         this.serialiser = createSerialiser();
-        this.resourceService = createResourceService();
-        this.auditService = createAuditService();
-        this.policyService = createPolicyService();
-        this.userService = createUserService();
-        this.cacheService = createCacheService();
-        this.palisadeService = createPalisadeService();
     }
 
     public Stream<T> read(final String filename, final String resourceType, final String userId, final String justification) {
-        Objects.requireNonNull(palisadeService);
+        Objects.requireNonNull(services.getPalisadeService());
```

Comment:
This shouldn't be required as the getPalisadeService() should do that check for you